### DISCUSSION TITLE
How to add support : Bseed (Led) Dimmer TS0601 _TZE200_3p5ydos3

### DISCUSSION THREAD

**Discussion by jvrrr** [Koenkk/zigbee2mqtt#11059] [Devices]
Hi,

I bought a BSEED dimmer, but it is not supported by Zigbee2Mqqt
So I read https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#instructions
And make some steps....But I got lost and I'm not able to do the next step.
Can some hero assist me to fix this?

This is what I have done so far:
https://github.com/Koenkk/zigbee2mqtt/issues/8911

I hoop to learn from it and assist with future new hardware.

Great thanks!

**Comment by Koenkk** [comment]
Let's continue in https://github.com/Koenkk/zigbee2mqtt/issues/8911
